MeesGame/Gameplay/GameManagement/GameEnvironment.cs

```diff
@@ -55,6 +57,11 @@ public static GameSettingsManager GameSettingsManager
         get { return gameSettingsManager; }
     }
 
+    public static GameEnvironment GetGameEnvironment
```

Comment:
GameEnvironment generates an error, if you have better suggestions, please let us know.